tests/Php81/fixtures/curl-echo-server.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+$output = $_FILES;
+foreach ($_FILES as $name => $file) {
+    unset($file['tmp_name']);
```

Comment:
Just in case, we should also check that the array has the proper structure.
Right now, if one submits `foo[]` as a file name, this file will leak the `tmp_name`.
Better not.

Reply:
I will work on this. I think we need to recursively iterate and remove the `tmp_name`. Probably worth adding an extra test too, for nested files.